### PR TITLE
feat: create TransferOnweship and PoolStrategyControl component

### PR DESCRIPTION
**Pull request by ChristanDaniel**
## Why
here components have been added to change the roles of the strategist and the pool owner, along with adapting the manager page for the strategist.

## Changes

- create ownership-claim page
- create TransferOnweship component
- create PoolStrategyControl component
- create ModalAlertTransferOwnership component

### CODE REVIEW

File: src/templates/OwnershipClaim/index.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+              fullWidth
+              disabledNoEvent={settingChain || !poolChainId}
+              onClick={() =>
+                setChain({
+                  chainId: `0x${poolChainId.toString(16)}`
+                })
+              }
+            />
+          ) : (
+            <Button
```
Comment (guilherme-correa-s) [inline src/templates/OwnershipClaim/index.tsx:108]: Would it be nice to have a button to connect?

File: src/templates/OwnershipClaim/index.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+              background="primary"
+              size="huge"
+              text="Claim"
+              fullWidth
+              disabledNoEvent={
```
Comment (guilherme-correa-s) [inline src/templates/OwnershipClaim/index.tsx:113]: What do you think about adding a message if the wallet doesn't match the candidate?

File: src/templates/PoolManager/Details/index.tsx [hunk 3]
```diff
@@ -32,6 +43,52 @@ const Details = () => {
...
+  const chainId = poolInfo ? poolInfo[0].chain_id : 0
+
+  const { txNotification, transactionErrors } = useTransaction()
+  const { setStrategist, transferOwnership } = useManagePoolController(
+    poolControllerAddress,
+    networks[chainId]?.rpc
+  )
+
+  async function handleChangeStrategy(address: string) {
+    const text = {
```
Comment (guilherme-correa-s) [inline src/templates/PoolManager/Details/index.tsx:56]: transactionText or transactionMessage, but it is just a suggestion

File: src/utils/substr.ts [hunk 4]
```diff
@@ -1,8 +1,8 @@
-function substr(address: string) {
+function substr(address: string, value = 4) {
```
Comment (guilherme-correa-s) [inline src/utils/substr.ts:1]: Just one more suggestion
```suggestion
function substr(address: string, indexEnd= 4) {
```
```suggestion
function substr(address: string, length= 4) {
```

**Review by Tiago138 — APPROVED** [review]
:+1:

File: src/constants/tokenAddresses.ts [hunk 5]
```diff
@@ -62,6 +62,9 @@ export const platform: Record<number, string> = {
   [137]: 'polygon-pos'
 }
 
+export const URL_KASSANDRA_CANDIDATE =
+  'https://www.kassandra.finance/ownership-claim'
```
Comment (guilherme-correa-s) [inline src/constants/tokenAddresses.ts:66]: It should be the link to the app, and it could only have one variable URL_KASSANDRA

**Review by guilherme-correa-s — APPROVED** [review]
LGTM

**Pull request merged**